Seed.Api/Controllers/UserController.cs

```diff
@@ -52,6 +52,25 @@ public async Task<IActionResult> Get(Guid userId)
 
             var result = new UserDto(user);
 
+            return Ok(result);
+        }
+
+                /// <summary>
+        /// Gets a user for the given email
+        /// </summary>
+        /// <param name="email" cref="string">Email of the user</param>
+        /// <response code="200">User for the given email</response>
+        /// <response code="404">User not found</response>
+        [HttpGet("ByEmail/{email}")]
```

Comment:
This is not a restfull address, add filters to the get all

Reply:
Removed

Reply:
either way if you need to search by id on the get all method you can always add a filter